token-metadata/0xB6eD7644C69416d67B522e20bC294A9a9B405B31/metadata.json

```diff
@@ -3,5 +3,20 @@
     "symbol": "0XBTC",
     "address": "0xB6eD7644C69416d67B522e20bC294A9a9B405B31",
     "decimals": 8,
-    "dharmaVerificationStatus": "VERIFIED"
```

Comment:
0x8432c31496EE917e914d327356240D1096f8b38a

Reply:
what is this address?